app/experimenter/experiments/constants/nimbus.py

```diff
@@ -156,25 +157,115 @@ class ApplicationConfig:
 )
 
 
+class Status(models.TextChoices):
+    DRAFT = "Draft"
+    PREVIEW = "Preview"
+    LIVE = "Live"
+    COMPLETE = "Complete"
+
+
+class PublishStatus(models.TextChoices):
+    IDLE = "Idle"
+    REVIEW = "Review"
+    APPROVED = "Approved"
+    WAITING = "Waiting"
+
+
+class LifecycleStates(Enum):
+    DRAFT_IDLE = dict(
+        status=Status.DRAFT,
+        publish_status=PublishStatus.IDLE,
+    )
```

Comment:
Why use this syntax instead of dict literal syntax? I don't think we use this anywhere else?